.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ripienaar
```

Comment:
while generally useful, I have to watch this repo for all changes.  This repo prouce 100s of emails per year, if not over a thousand. 

Since already watching the repo requesting review is redundant and will only add 100s of more emails per year to my inbox while adding no value.